### DISCUSSION TITLE
ExtraLoginOptions domain param vs dashboard whitelist

### DISCUSSION THREAD

**Discussion by qqqllppp** [Web3Auth/Web3Auth#364] [Help]
What is the difference between the `domain` param in `ExtraLoginOptions` of `connectTo` method (https://web3auth.io/docs/sdk/web/core/usage#extraloginoptions) vs the whitelist address on web3auth's dashboard?

**Comment by shahbaz17** [comment]
Hey @junhuang-ho 

These are the target domain for your proxy login providers like Auth0, Firebase, and Cognito, and the whitelist address is your application domain, such as the website integrating it. e.g. www.example.com

**Reply by qqqllppp** [reply]
Hi @shahbaz17 , I am unfamiliar with proxy login providers. For `domain` in the `ExtraLoginOptions`, what value should I put for production? So far in dev environment it is `localhost:3000`, but I assume it should be different in production? I am using firebase custom auth. Also say the domain of my website is `Moonlight.com`

**Reply by himanshuchawla009** [reply]
you can keep the domain values based on your env, for ex: http://localhost:3000 for your dev env and https://moonlight.com for prod env, easiest way in you case can be set domain: window.location.origin.

**Reply by qqqllppp** [reply]
What I mean is, is this `domain` param just for show/display purposes? Eg so that I can show that the redirected web3auth login is within my domain's brand? And that I can actually put whatever domain name I want, even if its not a registered domain?

**Reply by himanshuchawla009** [reply]
domain is used only for implicit oauth flow, firebase doesn't have implicit flow so the domain won't be used, but it is required in web3auth sdk coz of backward compatibility reasons, you can just pass it your window.location.origin , that is safe to do.
